src/main/java/com/connexta/transformation/api/ServiceRegistryQueueConstants.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformation.api;
+
+public abstract class ServiceRegistryQueueConstants {
```

Comment:
This class is never extended, should consider whether to keep it an abstract class

Reply:
Actually, I don't remember why I made that abstract... In any case, I don't really like having the class for constants. I'm going to look into just being able to throw those values in the application properties file.